### ISSUE TITLE
Error when installing controller

### ISSUE THREAD

**Issue by wildsky95** [NASA-Tensegrity-Robotics-Toolkit/NTRTsim#210]
When i install a controller to a model i get the error below
How can i fix it
![1491240450265-829020852](https://cloud.githubusercontent.com/assets/26694747/24621961/91a66f3a-18b8-11e7-914c-a9c53ae920db.jpg)

**Comment by apsabelhaus** [comment]
Hi wildsky,

Your image is blurry and I can't see what is going on. Please take a screenshot and upload that instead so we can help you!

Also, controllers aren't "installed" in NTRTsim. Please explain exactly what you mean, and exactly what you did, then we can help you figure it out.

Welcome to NTRT.
-Drew

**Comment by wildsky95** [comment]
thanks for your attention 
i coppied supperball example to the folder which i made in dev directrory then i deleted the supperball from the examples cmake file and add my folder to dev folders cmake list.after i added the controller on the Appsuperball and also i add the controller name to the executable files in cmake at mesioned directory then i wanted to compile the files by build.sh and i get this error which i attach the picture.
im researching on tensegrity models ill be grateful for your help
![screenshot from 2017-04-04 02-13-40](https://cloud.githubusercontent.com/assets/26694747/24633479/545cf8ec-18b8-11e7-9c60-d225bd63883a.png)
![screenshot from 2017-04-04 02-14-39](https://cloud.githubusercontent.com/assets/26694747/24633487/5a2ac98e-18b8-11e7-8cf7-baac2dda262b.png)
and also i cant run some of the executable files in build directory i run some of them and nothing happens,no simulation comes up ...!

**Comment by PerryBhandal** [comment]
I'm assuming you copied SUPERball from examples into your dev folder. Is
that correct? If you did, did you just copy some of the files but not
subfolders (such as controllers)?

**Comment by PerryBhandal** [comment]
Just saw the rest of your e-mail's body where you addressed part of my
question.

My question, restated: Did you copy the entire SUPERball folder (including
all subfolders) into /dev/amir?

**Comment by wildsky95** [comment]
Yes i coppies all and i also deleted the supperball from its cmake that it wont compile an just amir folder to be compiled because of the name issues i think,i just want to try adding some controllers to then add mine and work on it but i crash into this error

**Comment by PerryBhandal** [comment]
There's some change you're making beyond moving SUPERball to /dev/amir and updating the CMakeList which is causing your problem. Admittedly, that on its own isn't terribly helpful, but without seeing your changes it's difficult to specify what the exact issue is.

We can provide you more help if you can show us the exact changes you've made. The easiest way to do that is to go into your NTRTsim directory and run **git diff > changes.txt** then upload changes.txt.

**Comment by wildsky95** [comment]
sorry for the delay 
i dont understand what you mean by uploading changes.txt 
i want to mention this
i also tried the ready controllers which were in the Appsuperrball without any changes only i deleted // but i get the same error.im really stuck on this please help 
thanks to your attention

**Comment by wildsky95** [comment]
![screenshot from 2017-04-05 16-38-27](https://cloud.githubusercontent.com/assets/26694747/24704871/f992986c-19f8-11e7-9b28-784910a40e9b.png)
![screenshot from 2017-04-05 16-39-05](https://cloud.githubusercontent.com/assets/26694747/24704873/f99771ca-19f8-11e7-9122-f2d45726fb1c.png)
![screenshot from 2017-04-05 16-39-29](https://cloud.githubusercontent.com/assets/26694747/24704872/f996f6dc-19f8-11e7-8305-09e82ae137ab.png)
i uploaded the screenshots i made the changes which are just mentioned in the files itself why still i get this error
